### DISCUSSION TITLE
Is it possible to extend a page's background into the margins?

### DISCUSSION THREAD

**Discussion by amorphia** [spatie/laravel-pdf#146] [Q&A]
Hello hello,

I'm working on a multipage PDF but am running into an issue. We want to include a light gray background color for the pdf, which is easy enough with:

```css
html {
    -webkit-print-color-adjust: exact;
    background-color: #f9f9f9;
}
```
But we also want to use margins which we set via:

```php
PDF::view('pdf', $data)
    ->format(Format::Letter)
    ->margins(25, 0, 35, 0, Unit::Pixel);
```

But the background color set on the html element doesn't apply to the margin, and so everything looks off with these white gaps at the top and bottom of the page. 

If we don't use the margin method and instead just set a padding on the body of the content the background looks fine, but the content butts right up to the top and bottom of the page on page breaks which looks gross, and we don't have room to place a pagecount footer.

Any suggestions on how to resolve this?

**Comment by jsandfordhughescoop** [comment]
Having this exact issue as well
